EIPS/eip-6963.md

```diff
@@ -268,22 +195,28 @@ function onPageLoad() {
 }
 ```
 
-### Ethereum library
+### DApp implementation
 
-Here is a reference implementation for a Ethereum library to display and track multiple wallet providers that are injected by browser extensions.
+Here is a reference implementation for a DApp to display and track multiple wallet providers that are injected by browser extensions. 
 
 ```typescript
-function onPageLoad() {
-  const providers: EIP6963ProviderDetail[] = [];
+function onPageLoad(): EIP6963ProviderDetail[] => {
```

Comment:
```suggestion
function onPageLoad() => {
```

Return types are inferred in TypeScript.

Reply:
Nit: Shouldn’t it ideally return a cleanup function if anything though? To clean up the `addEventListener`

Reply:
Potentially, but it's not required so that would be up to an implementer to decide if they want to take that approach. My goal with these changes is to rely upon the text to decide what should be done not the examples. The examples, are there more to make the text a bit more clear.

Reply:
Makes sense!